### PR TITLE
New CFE workflow GitHub template

### PR DESCRIPTION
**Pull request by naman03malhotra**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Moved here from PR: https://github.com/woocommerce/woocommerce-quality/pull/913

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-cfe-template.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    id: plan-if-defects-discovered
+    attributes:
+      label: "What is the plan should defects to be discovered in these PRs ahead of final release?"
+      placeholder: "Reverting this PR and ... would be enough"
+    validations:
+      required: true
+  - type: textarea
+    id: how-to-communicate
+    attributes:
+      label: "How should this change be communicated in the release post on the [public developer blog](https://href.li/?https://developer.woo.com/blog/)?"
```
Comment (albarin) [inline .github/ISSUE_TEMPLATE/new-cfe-template.yml]: Nit: while testing this template I noticed the md link is not working on the `label` attribute:

Screenshot 2024-09-18 at 12 12 45

Maybe we can add the link in the `description` field, it seems to work well there:

Screenshot 2024-09-18 at 12 12 18

Something similar to:

```md
- type: textarea
    id: how-to-communicate
    attributes:
      label: "How should this change be communicated in the release post on the public developer blog:"
      description: "See the blog at [https://developer.woo.com/blog/](https://developer.woo.com/blog/)"
      placeholder: "There is no need to add new communication to the already planned one."
    validations:
      required: true

**Review by albarin — APPROVED** [review]
Looking good! Pre-approving!

Just added a small comment regarding link formatting.

File: .github/ISSUE_TEMPLATE/new-cfe-template.yml (hunk 1 above)
Comment (naman03malhotra) [reply .github/ISSUE_TEMPLATE/new-cfe-template.yml]: Good catch, fixed it here: https://github.com/woocommerce/woocommerce/pull/51468/commits/f819e7942992342345b2aa838f717015683fe28a

**Comment by Konamiman** [comment]
This is an oversight on my side, but the issue body says:

> The request will be reviewed and accepted or denied:
>        
>  * If accepted, a pull request targeting the appropriate release branch will be created.
>  * If rejected, a pull request targeting `trunk` will be created.

This wording comes from the initial version of the template, where it asked for a branch name. Now that it asks for PR URLs instead, this should be more like "If accepted, the pull requests will be included in the requested version; if denied, they will be processed as a regular pull request".

**Comment by naman03malhotra** [comment]
>This wording comes from the initial version of the template, where it asked for a branch name. Now that it asks for PR URLs instead, this should be more like "If accepted, the pull requests will be included in the requested version; if denied, they will be processed as a regular pull request".

Good catch, I've updated the copy accordingly.

**Review by Konamiman — APPROVED** [review]

**Pull request merged**